rules-reviewed/camel3/camel2/tests/data/java-generic-information/camelContextDeprecatedMethodsBlueprint.xml

```diff
@@ -0,0 +1,11 @@
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+
+<camelContext id="camel1" xmlns="http://camel.apache.org/schema/spring">
```

Comment:
I think this should be the blueprint schema

Reply:
@m-brophy fixed, thanks !